files/katex/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "bower",
```

Comment:
Whoops, I missed you were using `bower`. Can you let me know if its good to merge @as-com?